.solcover.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
         "rounds/AdjustableRoundsManager.sol",
         "pm/mixins/interfaces",
         "bonding/deprecated",
+        "token/ArbitrumLivepeerToken.sol" // testnet only
```

Comment:
Since this is just a testnet contract, I went light on tests for now and removed this from coverage.